tests/test_jump_cas22.py

```diff
@@ -447,7 +452,7 @@ def test_fit_ramps(detector_data, use_jump, use_dq):
 
             assert use_dq # sanity check that this branch is only encountered when use_dq = True
 
-    chi2 /= np.sum(okay)
+    chi2 /= (np.sum(okay) + excess)
```

Comment:
I don't think it makes a difference, but I don't think I expected the "excess" to make an appearence here, since those cases don't contribute to chi^2 above on line 444.

Reply:
It actually should be be a difference not a sum, but you are right, it makes no difference

Reply:
Yup, would make sense subtracted, thanks.